### PR TITLE
Fix #176432 biolink.webtena.com

### LINKED ISSUE #176432: biolink.webtena.com
### Issue URL (Anti Adblock Script)
 [https://biolink.webtena.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbiolink.webtena.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.31
System version: | windows11
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances

### PR DESCRIPTION
**Pull request by SpotlightForBugs**
## Fix #176432 

**Pull Request Summary:**
Resolved issue [#176432]  by implementing a solution that is stopping the anti-adblock script before it activates.

**Checklist for a Valid Pull Request:**
- [x] This submission is neither an advertisement nor a bug report.
- [x] My contribution adheres to the project's [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [filter syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters).
- [x] I have thoroughly reviewed my changes.
- [x] My updates maintain the integrity of websites, applications, and file structures.

**Problem Addressed by This Pull Request:**
The pull request resolves an issue [#176432] where AdGuard was being detected on websites, potentially leading to functionality disruptions. 

<details>
<summary>Screenshots</summary>

image

image

</details>

**Additional Notes:**

**Similar Pull Request Notice:**
>This pull request is similar to a previous fix for issue [#176383](https://github.com/AdguardTeam/AdguardFilters/issues/176383), which addressed a timeout prevention mechanism using the presence of this code: 
https://github.com/AdguardTeam/AdguardFilters/issues/176383#issuecomment-2040012679
image

- [ ] Unblocked advertisements or residual ad elements.
- [ ] Malfunctioning websites or applications.
- [x] Detection of AdGuard on a website.
- [ ] Overlooked analytics or trackers.
- [ ] Social media integrations such as share, like, or tweet buttons.
- [ ] General annoyances including pop-ups and cookie notifications.
- [ ] Routine filter maintenance.

**Consent:**
- [x] I confirm that this pull request is free of private information and meets all outlined conditions.

### CODE REVIEW

**Review by Alex-302 — CHANGES_REQUESTED** [review]
This anti-adblock blocks using of the site.
Correct place - https://github.com/AdguardTeam/AdguardFilters/blob/5136e7865d95a5882c81da8aa35795bea7e236f8/BaseFilter/sections/antiadblock.txt#L1073

File: AnnoyancesFilter/Popups/sections/antiadblock.txt [hunk 1]
```diff
@@ -3105,7 +3105,6 @@ animenewsnetwork.com#@##Adbanner
 @@||yandex.ru/ads/system/context.js$xmlhttprequest,domain=sdamgia.ru
 timeanddate.com#@##ad-wrap2
 lite.liiingo.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
-biolink.webtena.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/antiadblock.txt]: Both rules are ok) Just add to https://github.com/AdguardTeam/AdguardFilters/blob/5136e7865d95a5882c81da8aa35795bea7e236f8/BaseFilter/sections/antiadblock.txt#L1075

```
! https://github.com/AdguardTeam/AdguardFilters/issues/176432
biolink.webtena.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
!+ PLATFORM(ios, ext_android_cb)
@@||biolink.webtena.com^$generichide
```

Comment (SpotlightForBugs) [reply AnnoyancesFilter/Popups/sections/antiadblock.txt]: lite.liiingo.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
is equally misplaced

**Review by SpotlightForBugs — COMMENTED** [review]
I am sorry, do you need anything else to merge?

File: BaseFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -1072,7 +1072,14 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 !
 ! NOTE: Regular rules
 !
-!
+! https://github.com/AdguardTeam/AdguardFilters/issues/176432
+biolink.webtena.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
+!+ PLATFORM(ios, ext_android_cb)
+@@||biolink.webtena.com^$generichide
+!https://github.com/AdguardTeam/AdguardFilters/issues/176383
```
Comment (Alex-302) [inline BaseFilter/sections/antiadblock.txt:1079]:
```suggestion
! https://github.com/AdguardTeam/AdguardFilters/issues/176383

```
Please add a space after `!` in comments (not ditectives or hints)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/antiadblock.txt (hunk 2 above)
Comment (SpotlightForBugs) [reply BaseFilter/sections/antiadblock.txt:1079]: I've added the space

**Review by Alex-302 — APPROVED** [review]

**Comment by D4niloMR** [comment]
Maybe a generic fix? `https://www.google.com/search?&q=%22We+serve+ads+so+we+can+keep+our+website+running.+Please+disable+your+ad+blockers.%22`

I tested in AdGuard, and these filters from uBO work fine

https://github.com/uBlockOrigin/uAssets/blob/master/filters/filters-2023.txt#L1582-L1589
```
#@#.banner-ads
#@#.banner_ads
#@#.ad-zone
#@#.ad-space
##.banner-ads:not(.textads)
##.banner_ads:not(.textads)
##.ad-zone:not(.textads)
~nieru.net##.ad-space:not(.textads):not(.adbox)
```

from chp adblock:

https://github.com/uBlockOrigin/uAssets/blob/master/filters/filters-2021.txt#L5123
https://github.com/uBlockOrigin/uAssets/blob/master/filters/filters-2021.txt#L5125
```
#@#.ad-unit
~lamire.jp##.ad-unit:not(.text-ad):not(.textads)
```

**Pull request merged**